devtools/gradle/gradle-model/src/main/java/io/quarkus/gradle/tooling/GradleApplicationModelBuilder.java

```diff
@@ -28,7 +28,6 @@
 import org.gradle.api.tasks.SourceSet;
 import org.gradle.api.tasks.SourceSetContainer;
 import org.gradle.api.tasks.compile.AbstractCompile;
-import org.gradle.composite.internal.CompositeProjectComponentArtifactMetadata;
 import org.gradle.internal.composite.IncludedBuildInternal;
```

Comment:
@aloubyansky @glefloch isn't this class internal too? Wondering if there are alternatives.

Reply:
It is. It's not obvious how to avoid it for me. Ideally, we don't want to rely on internal API. @glefloch if you have some time to research this one, it'd be very much appreciated.

Reply:
yes I will have a look on it.